source/Ship.cpp

```diff
@@ -702,7 +713,10 @@ void Ship::FinishLoading(bool isNewInstance)
 		}
 		else
 			++it;
-		if(bay.side == Bay::INSIDE && bay.launchEffects.empty() && Crew())
+
+		if (removeBays)
+			it = bays.erase(it - 1);
```

Comment:
You should remove all bays, not just one.

Reply:
removeBays is always true, then it = bays.erase(it - 1); will always get executed, no?

Reply:
it removes per iteration, preping the code for partial removal

Reply:
@Hurleveur  in the load part you mean?

Reply:
I'll be honest I'm not sure, I'll just let warp-core handle it from now, since this was his idea.
quoting him from discord: `Setting the bool is good, but I think it should be used in FinishLoading() when importing the bays from the base class, not in the form it's currently being used, and not only for inside bays.`

Reply:
I understand.

out of topic, I don't game much, thus no discord (decided to look it up now)